resources/dicts/patrols/general/medcat.json

```diff
@@ -279,5 +279,644 @@
+        null,
+        "s_c seems a little distracted, deep in thought about something. When r_c asks what's up, s_c blinks slowly at them before confessing how much they miss spending time together. They burrow their noses into each others pelts and purr, soaking in the moment and their love for each other before continuing on, and promise to spend more time together. "
+    ],
+    "fail_text": [
+        "p_l and r_c are far too distracted catching up with one another, and get nothing useful done.",
+        "s_c seems a little distracted, deep in thought about something. When r_c asks what's up, s_c blinks slowly at them before confessing how much they miss spending time together. r_c tries to comfort them, but s_c shakes their head, knowing that no matter what they say, their roles are just too different. Both cats are too upset to gather herbs."
+    ],
+    "win_trait": [
+        "loving"
+    ],
+    "fail_trait": [
+        "lonesome",
+        "insecure"
+    ],
+    "relationship_constraint": [
+        "siblings",
+        "platonic_20"
+    ],
+    "min_cats": 2,
+    "max_cats": 2,
+    "antagonize_text": null,
+    "antagonize_fail_text": null
+},
+{
+    "patrol_id": "gen_med_medwarriorapp_curiosity1",
+    "biome": "Any",
+    "season": "Any",
+    "tags": [
+        "med_cat",
+        "warrior_apprentice",
+        "one_apprentice",
+        "random_herbs",
+        "respect",
+        "comfort",
+        "trust",
+        "platonic"
+    ],
+    "intro_text": "p_l leads app1 through c_n's territory, enjoying the nice day. They glance back, and realize app1 looks like they want to ask them a question.",
+    "decline_text": "Just as app1 opens their mouth to speak, p_l spots a Clanmate trying to remove a thorn from their paw over their shoulder, and they hurry away to help.",
+    "chance_of_success": 50,
+    "exp": 20,
+    "success_text": [
+        "p_l sits down and motions for app1 to join them, looking at them inquisitively. app1 tentatively explains that they're curious about herbs, but are worried that desire to learn is detrimental. p_l reassures them that nobody would think less of them for that, and it's good to know the basics. p_l promises to teach them when they get back to camp.",
+        "p_l sits down and motions for app1 to join them, looking at them inquisitively. app1 studiously looks at their paws, tail curled around their leg in shame, and mutters that they don't want to talk about it. p_l gently reassures them that it's okay to ask questions, and though app1 doesn't ask, they seem more chipper, and they're a big help anyways."
+    ],
+    "fail_text": [
+        "p_l sits down and motions for app1 to join them, looking at them inquisitively. app1 studiously looks at their paws, tail curled around their leg in shame, and mutters that they don't want to talk about it. The apprentice unfortunately isn't much help from then on, and the patrol returns home empty-pawed."
+    ],
```

Comment:
i wonder abt a potential fail stat outcome that centers around a cold or strict med cat ignoring the app's behavior?  also wonder if the 1st win outcome would be better suited as a win stat outcome for a med cat with empathetic/compassionate/other benevolent traits.

This could also be applicable to the other curiosity patrol